### ISSUE TITLE
Is there a way to limit how much information an item can store?

### ISSUE THREAD

**Issue by nunovasquescambra** [tiberiuzuld/angular-gridster2#688]
Sorry if the question is not worded properly...

`  dropItem(dragId: string): void {
    const { components } = this;
    const comp: IComponent = components.find(c => c.id === this.dropId);
    const updateIdx: number = comp ? components.indexOf(comp) : components.length;
    const componentItem: IComponent = {
      id: this.dropId,
      componentRef: dragId
    };
    this.components = Object.assign([], components, { [updateIdx]: componentItem });
  }`

Basically, my code is creating UUIDs for items in the grid and components being thrown inside them.

I'm trying to create some type of logic that ensures only one component can be stored inside each item...

Thank you for your time in advance!

Any hints will be really appreciated :)
